metricbeat/module/kubernetes/_meta/test/README.md

```diff
@@ -16,3 +16,18 @@ path: "../_meta/test"
 ```
 
 Since all metricsets use a `data.json` file for the documentation - this file contains an example of the metrics fields -, **it is necessary that one of these files has the name `docs.plain`**. This file should have the **same content as one of the metrics file of a KSM version** we support. This means that one of the files is duplicated, for example: `ksm.v2.7.0.plain` has the same content as `docs.plain`. This is not a mistake, as having the `ksm.v2.7.0.plain` tells us the metrics of that specific version.
+
```

Comment:
How did that error block ended up in the Readme?

Reply:
Oops that was an accident. I was running the tests and I ran into that error, and I was thinking of updating the README in case someone runs into the same in the future. But I deleted it now, and I will open a new PR in case I decide to update the README file to not make this PR bigger than it already is.